src/pages/Home/Home.styled.jsx

```diff
@@ -51,20 +119,31 @@ export const Button = styled(Link)`
   text-align: center;
   text-decoration: none;
   line-height: 40px;
+  font-weight: bold;
+
+  position: relative;
+  transform: translate(0%, 1100%);
 
   @media (min-width: 960px) {
-    margin-top: -35%;
+    transform: translate(0, 0);
+    position: static;
     background-color: var(--secondary);
-    width: 45%;
+    height: 50%;
+    width: 100%;
+    padding: 1rem 0 3rem 0;
+    font-weight: bold;
+    font-size: 1.5rem;
+
+    border-radius: 2rem;
+    letter-spacing: 2px;
+    transform: translate(0, 0);
   }
-`
 
-export const Logo = styled.img`
-  display: none;
+  @media (min-height: 700px) and (min-width: 375px) and (max-width: 600px) {
+    transform: translate(0%, 1200%);
```

Comment:
WFT with those values? 😮

Reply:
this works with the absolute position and the flex reverse